tests/System.Text.JsonLab.Tests/ReadFromPipeTests.cs

```diff
@@ -1,232 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using Newtonsoft.Json;
-using System.Buffers;
-using System.IO;
-using System.IO.Pipelines;
-using System.Text.JsonLab.Tests.Resources;
-using System.Threading.Tasks;
-using Xunit;
-
-namespace System.Text.JsonLab.Tests
-{
-    public class ReadFromPipeTests : IDisposable
```

Comment:
FYI @tdykstra
This is roughly what reading from Pipes would look like. We'd want to re-write this against the final API shape that we shipped with (this test in corefxlab was written against the prototype code/API). I will re-write this and share the sample this week (just a heads up in case you were interested in giving it a shot).

The write side is in WriteToPipTests.cs.